smokefree/sites/all/themes/smokefreegov/js/tophat.js

```diff
@@ -7,4 +7,14 @@ jQuery(document).ready(function() {
     if (jQuery(".scrollmenu").length > 0) {
         jQuery('#main-nav').css('margin-top', "49px");
     }
```

Comment:
Do you mind adding comments here?

Thanks!

Reply:
comments added.